src/components/Carousel.tsx

```diff
@@ -21,14 +21,11 @@ const Inner = styled.div({
   gridAutoFlow: "column",
 });
 
-const WrappedLink = styled(motion(Link))({
+const WrappedLink = styled(Link)({
   width: "var(--card-size)",
+  height: "var(--card-size)",
```

Comment:
the height needs to be dynamic, images are all different heights

Reply:
Ah right.. Not having a fixed height meant they were warping weirdly, but might be something to do with [this](https://www.framer.com/motion/layout-animations/###the-content-stretches-undesirably).